src/esthri/src/config.rs

```diff
@@ -0,0 +1,73 @@
+use once_cell::sync::OnceCell;
+use serde::Deserialize;
+
+// This is the default chunk size from awscli
+const CHUNK_SIZE: u64 = 8 * 1024 * 1024;
+const XFER_COUNT: usize = 16;
```

Comment:
perhaps XFER_WORKER_COUNT to imply this is parallelism.  Or just `MAX_WORKERS` as likely this same number will be used as default for any other non-xfer parallelism.